pydantic/functional_validators.py

```diff
@@ -170,15 +198,32 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: _GetCoreSchema
         except PydanticSchemaGenerationError:
             serialization = None
 
+        input_schema = (
+            None
```

Comment:
Regarding your question from above:

>  Decide if we want to default to Any for input_type (so that the JSON Schema for a model with a PlainValidator works out of the box).

I'd opt for yes - we basically infer this under the hood, so I'd opt into exposing this in JSON schema as well.

Reply:
Made `Any` the default only for `PlainValidator`. For the others, the current behavior is to generate a JSON Schema for the type annotation (i.e. in related tests, `int`). Overriding this with `Any` would be a breaking change. I updated the tests if you want to see how it behaves.